File: binary_calculator_js/calculate.js
function calculate(n1, n2, o) {
  switch (o) {
    case 'add':
      return (parseInt(n1, 2) + parseInt(n2, 2)).toString(2)
    case 'subtract':
      return (parseInt(n1, 2) - parseInt(n2, 2)).toString(2)
    case 'multiply':
      return (parseInt(n1, 2) * parseInt(n2, 2)).toString(2)
  }
}
module.exports = calculate;

File: python/binary_calculator/calculate.py
def calculate(n1, n2, o):
    match o:
        case "add":
            return bin(int(n1, 2) + int(n2, 2))[2:]
        case "subtract":
            res = bin(int(n1, 2) - int(n2, 2))
            return res.replace("0b", "")
        case "multiply":
            return bin(int(n1, 2) * int(n2, 2))[2:]
